gulpfile.js/tasks/setBase.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (config) {
   developmentBase += '\ndocument.write(\'<base href="\' + theBaseUrl + \'"/>\');'
   developmentBase += '\n</script>'
 
-  const prodUrl = config.env.prod.baseUrl
+  const prodUrl = process.env.HUGO_BASE_URL || config.env.prod.baseUrl
```

Comment:
yes. this is used for some json requests. if you want you can remove it later and test all the existing functionality